presto-docs/src/main/sphinx/presto_cpp/properties-session.rst

```diff
@@ -419,3 +419,24 @@ writer scaling if it is detected as overloaded by scale writer exchange.
 
 Minimum amount of data processed by all the logical table partitions to
 trigger skewed partition rebalancing by scale writer exchange.
+
+``native_table_scan_scaled_processing_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Enables scaled processing for table scans.
```

Comment:
maybe clarify that this is scanning the concurrency per-worker, and not scaling the number of workers.